tests/components/admin_console/jobs/__snapshots__/table.test.jsx.snap

```diff
@@ -0,0 +1,407 @@
+          key="1235"
+        >
+          <td
+            className="whitespace--nowrap text-center"
+            width="30px"
+          />
+          <td
+            className="whitespace--nowrap"
+          >
+            <span
+              className="status-icon-error"
+              title="Job ID: 1235"
+            >
+              <FormattedMessage
+                defaultMessage="Canceled"
+                id="admin.jobTable.statusCanceled"
+                values={Object {}}
+              />
+            </span>
+          </td>
+          <td
+            className="whitespace--nowrap"
+          >
+            <span
+              className="whitespace--nowrap"
+            >
+              <FormattedDate
+                day="2-digit"
+                month="short"
+                value={2018-10-29T17:31:34.674Z}
+                year="numeric"
+              />
+               - 
+              <FormattedTime
+                hour="2-digit"
+                minute="2-digit"
+                value={2018-10-29T17:31:34.674Z}
+              />
+            </span>
+          </td>
+          <td
+            className="whitespace--nowrap"
+          >
+            <span
+              className="whitespace--nowrap"
+            >
+              --
+            </span>
+          </td>
+          <td
+            colSpan="3"
+          >
+            <span />
+          </td>
+        </tr>
+        <tr
+          key="1236"
+        >
+          <td
+            className="whitespace--nowrap text-center"
+            width="30px"
+          />
+          <td
+            className="whitespace--nowrap"
+          >
+            <span
+              className="status-icon-error"
+              title="Job ID: 1236"
+            >
+              <FormattedMessage
+                defaultMessage="Error"
+                id="admin.jobTable.statusError"
+                values={Object {}}
+              />
+            </span>
+          </td>
+          <td
+            className="whitespace--nowrap"
+          >
+            <span
+              className="whitespace--nowrap"
+            >
+              <FormattedDate
+                day="2-digit"
+                month="short"
+                value={2018-10-29T17:31:34.674Z}
+                year="numeric"
+              />
+               - 
+              <FormattedTime
+                hour="2-digit"
+                minute="2-digit"
+                value={2018-10-29T17:31:34.674Z}
+              />
+            </span>
+          </td>
+          <td
+            className="whitespace--nowrap"
+          >
+            <span
+              className="whitespace--nowrap"
+              title="Last Activity: Oct 29, 2018 - 11:01:34 PM"
+            >
+              NaN minutes
+            </span>
+          </td>
+          <td
+            colSpan="3"
+          >
+            <span />
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</div>
+`;
```

Comment:
Snapshot test is failing due to time difference.  Maybe need to change something from the JobTable component, like:
- use formatDate/formatTime function instead of FormattedDate/FormattedTime components.  Then on test, mock those function to return whatever is passed as a date/time.

Reply:
Using a fixed timezone for tests now. So, mocking is not needed